platform/accessibility/common-issues/focus-order-guidelines.md

```diff
@@ -0,0 +1,70 @@
+We build a lot of rich interet applications (RIA) using React, for VA.gov. We often update routes using client-side technology, add and remove HTML fragments, and update interfaces without making page requests to a server.
+
+These type of client-side changes require us to think about focus order. 
+
+- What elements can receive keyboard focus when users press the `TAB` key or the `SHIFT + TAB` chord?
+- Are there non-focusable elements that we want to receive keyboard focus?
+- Are there elements that need to receive focus when a dynamic event like a data fetch is completed?
+
+This is a common issue that can be solved with human-centered design and a collaborative effort among design, front-end engineering, and accessibility.
+
+## [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)
+Applications must have elements that receive [focus in an order that preserves meaning and usability](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html) for sighted, keyboard, and assistive technology users. All users should be able to determine where their keyboard focus is quickly, and understand when that focus changes based on changes in the user interface.
+
+## Acceptance Criteria
+
+- [ ] As a visual, keyboard, or screen reader user, I want to understand the focus order and navigate your web application successfully.
+- [ ] As a screen reader user, I want to hear partial page changes or dynamic updates announced properly
+- [ ] As a keyboard user, I want focus to be maintained or reset in a predictable way when a dynamic update occurs
+
+## Environment
+
+* Operating System: all
+* Browser: all
+* Screenreading device: all
+* Server destination: all
+
+## Steps to Confirm Current Focus
+
+1. Enter [URL] in browser (Chrome works well)
+1. Right click on the page and click `Inspect` to open Chrome dev tools
+1. Click `Console` in the Dev Tools palette
+1. Type the following snippet into your console to confirm keyboard focus<br/><br/>
+    ```javascript
+    document.activeElement
+    ```
+1. Ensure focus is in line with design guidance or review with your accessibility specialist
+
+## Possible Fixes (optional)
+There are two situations where adding a `tabindex` attribute is preferable: When you need to set focus with JavaScript, or when you need to add a non-focusable element to the document focus order.
+
+### Setting focus with JavaScript
+Our rich applications often require us to update small sections of the page, or use client-side routing to change views. These are both good candidates for setting focus with JavaScript. We do this to help screen readers understand that something has changed on the page, and to avoid extra tab stops for keyboard users. Elements that will receive keyboard focus via scripting must have a `tabIndex="-1"` attribute.
```

Comment:
In addition to helping screen readers, would it be helpful to note that communicating the context change supports people with cognitive considerations? Since that is such a large part of our audience, I think it underscores the relevance of communicating focus and context change.